src/services/statisticMail/hooks/index.js

```diff
@@ -0,0 +1,28 @@
+const hooks = require('feathers-hooks-common');
+const { authenticate } = require('@feathersjs/authentication');
+const globalHooks = require('../../../hooks');
+
+
```

Comment:
Codacy found an issue: [More than 2 blank lines not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5768742)